cmd/common.go

```diff
@@ -602,15 +605,335 @@ func exportCodesignFiles(tool Tool, archivePath, outputDirPath string) error {
 		return err
 	}
 
+	provProfilesUploaded := (len(profilesToExport) == 0)
+	certsUploaded := (len(certificatesToExport) == 0)
+
+	if len(profilesToExport) > 0 || len(certificatesToExport) > 0 {
+		fmt.Println()
+		shouldUpload, err := askUploadGeneratedFiles()
+		if err != nil {
+			return err
+		}
+
+		if shouldUpload {
+			var err error
+			provProfilesUploaded, certsUploaded, err = uploadExportedFiles(profilesToExport, certificatesToExport, outputDirPath)
+			if err != nil {
+				return err
+			}
+
+		}
+
+	}
+
 	fmt.Println()
 	log.Successf("Exports finished you can find the exported files at: %s", outputDirPath)
+
 	if err := command.RunCommand("open", outputDirPath); err != nil {
 		log.Errorf("Failed to open the export directory in Finder: %s", outputDirPath)
 	} else {
 		fmt.Println("Opened the directory in Finder.")
 	}
 
-	printFinished(certificatesOnly)
+	printFinished(provProfilesUploaded, certsUploaded)
+
+	return nil
+}
+
+func uploadExportedFiles(profilesToExport []profileutil.ProvisioningProfileInfoModel, certificatesToExport []certificateutil.CertificateInfoModel,
```

Comment:
do not use named returns values in this long func

Reply:
We need it here because this func returns 2 bool attr. and it can cause a misconception.
And if the provisioning profiles are uploaded successfully but the certs. are not, only the first bool (provProfilesUploaded) will be true.